dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java

```diff
@@ -405,4 +410,57 @@ public static long countFailures(List<ArtifactResults> table, String columnName)
         .filter(row -> row.getResult(columnName) == null || row.getFailureCount(columnName) > 0)
         .count();
   }
+
+  private static final int SUMMARIZING_DEPENDENCY_PATH_SIZE_THRESHOLD = 5;
```

Comment:
This constant name is still unclear. Perhaps MINIMUM_NUMBER_DEPENDENCY_PATHS?

Reply:
Updated